tool/client.cc

```diff
@@ -378,18 +453,86 @@ static void InfoCallback(const SSL *ssl, int type, int value) {
   }
 }
 
-bool Client(const std::vector<std::string> &args) {
-  if (!InitSocketLibrary()) {
-    return false;
+static int verify_cb(int ok, X509_STORE_CTX *ctx)
+{
+  X509 *err_cert;
+  int err, depth;
+  BIO* bio_err = BIO_new_fp(stdout, BIO_CLOSE);
```

Comment:
Should this point to `stdout` or `stderr`? Only asking because the `bio_err` naming seems to indicate `stderr`, ~~but no strong opinions on which should be used here.~~ Rest of the code seems to be using `stdout`

Reply:
you are correct, should be stderr - changed it :)